src/VirtoCommerce.SimpleExportImportModule.Web/Content/css/simple-export-import.css

```diff
@@ -79,3 +83,35 @@
 .simple-export-import .align-items-center {
     align-items: center;
 }
+
+.simple-export-import .label {
+    font: bold 14px Roboto;
+    margin-bottom: 5px;
+}
+
+.simple-export-import .count {
+    font: bold 14px Roboto;
+}
+
+.simple-export-import .radio-label {
+    font: normal 13px Roboto;
+    color: #96989C;
+    margin-right: 1rem;
+}
+
+.simple-export-import .mr-auto {
+    margin-right: auto;
+}
+
+
+.simple-export-import .h-100 {
+    height: 100%;
+}
+
+.simple-export-import .p-3 {
+    padding: 1rem;
```

Comment:
I'm not sure that we need to use REM's here (and above), since we don't have bootstrap here and because of that 'p-3' class doesn't give an idea of ​​how much padding we are adding